telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex

```diff
@@ -0,0 +1,32 @@
+defmodule TelemetryApi.Periodic.OperatorFetcher do
+  use Task
+  alias TelemetryApi.Operators
+
+  wait_time_str = System.get_env("OPERATOR_FETCHER_WAIT_TIME_MS") ||
+    raise """
+    environment variable OPERATOR_FETCHER_WAIT_TIME_MS is missing.
+    """
+
+  @wait_time_ms (
+    case Integer.parse(wait_time_str) do
+      :error -> raise("OPERATOR_FETCHER_WAIT_TIME_MS is not a number")
```

Comment:
Might be a good idea to include the actual value.

Reply:
Yep, that would help